docs/include/conf/virtual/talk-recording-guide.rst

```diff
@@ -0,0 +1,136 @@
+This page lists the requirements and best practices for recording your talk in advance, so that we can maintain consistent quality and streaming stability across all talks.
+
+Since we are aware that not everyone is experienced or comfortable with recording at home, we're happy to share some guidelines and recommendations for creating the best talk video possible.
+
+If you don't feel comfortable recording at all, you have the option to schedule a "private stream" session with our video producer, where you can present your talk live over video conference, and they will record your camera and slides for later processing. Please email us to request a private stream session.
+
+All talks are going to be processed and uploaded in advance, so it's especially critical that you complete your recording on time! Please email us if you have any questions.
+
+All visual and audio content (including videos, room background, and slides) must adhere to the Write the Docs Code of Conduct.
+
+Video requirements
+------------------
+
+- Duration: 30min (+- a few minutes is ok)
+- No cutting required if the video is in a single file
```

Comment:
I am confused by what this means. Especially because later on, we say we can cut parts out of the video if needed:
> While you are recording the talk, if you run into a problem with your narration but you don’t want to stop recording, we can edit it later! Take a breath, repeat the last 2-3 sentences, and when you send us the video file please write down time markers of places where edits are needed.

Reply:
Hmm, well the point is that if they are recording a continous single file they don't need to trim the lead-in and lead-out footage (because we instruct them to wait 10-20 seconds before and after the talk). Feel free to suggest a different phrasing, maybe we can replace "cutting" with "editing"?

Reply:
@mxsasha what do you think about my suggestion?

Reply:
Perhaps:

```suggestion
- Please send a single video file without edits
```

Reply:
I like Eric's suggestion. The current text makes me think the opposite: that the file I deliver as a speaker must not require any further cutting.